avr_demo/run_suite.py

```
@@ -0,0 +1,180 @@
+
+def get_depths_from_build_rs():
+    """Parses build.rs to extract the DEPTHS constant."""
+    try:
+        with open("build.rs", "r") as f:
+            content = f.read()
+            match = re.search(r"const DEPTHS: &\[usize\] = &\[(.*?)\];", content, re.DOTALL)
+            if match:
+                depths_str = match.group(1).replace('\n', '').replace(',', ' ').split()
+                return [int(d) for d in depths_str]
+    except (IOError, ValueError) as e:
+        print(f"Could not read or parse DEPTHS from build.rs: {e}", file=sys.stderr)
```

Ruff (0.11.9)

Undefined name `sys`

(F821)